docs/docs/documentation/getting-started/quick-start.md

```diff
@@ -65,6 +65,11 @@ solace-agent-mesh build
 You must run the `solace-agent-mesh` commands at the root directory of your project where the `solace-agent-mesh.yaml` file is located.
 :::
 
+:::warning
```

Comment:
I would prefer to remove this tip for now. We can add it back if we get feedback that multiple people are tripping over this.